web/api/queries/countif/README.md

```diff
@@ -0,0 +1,34 @@
+<!--
+title: "CountIf"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/web/api/queries/countif/README.md
+-->
+
+# CountIf
+
+> This query is available as `countif`.
+
+CountIf returns the percentage of points in the database that satisfy the condition supplied.
+
+The following conditions are available:
+
+- `!` or `!=` or `<>`, different than
+- `=` or `:`, equal to
+- `>`, greater than
+- `<`, less than
+- `>=`, greater or equal to
+- `<=`, less or equal to
+
+The target number and the desired condition can be set using the `group_options` query parameter, as a string, like in these examples:
+
+- `!0`, to match any number except zero.
+- `>=-3` to match any number bigger or equal to -3.
+
+## how to use
+
+This query cannot be used in alarms.
+
+`countif` changes the units of charts. The result of the calculation is always from zero to 1, expressing the percentage of database points that matched the condition. 
```

Comment:
I am observing that sometimes I am having `NULL` values in some columns and I always have `NULL` for the most recent value returned when `group_options` is not given:

```sh
$ curl -k https://localhost:19999/api/v1/data?chart=apps.mem&group=countif
{
 "labels": ["time","netdata","apps.plugin","freeipmi.plugin","nfacct.plugin","python.d.plugin","tc-qos-helper","go.d.plugin","ebpf.plugin","httpd","sql","wifi","modem","netmanager","logs","nfs","ssh","dhcp","cron","X","ksmd","system","kernel","other","VMs","time"],
    "data":
 [
 [ 1655689520,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],
 [ 1655689519,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,100],
 [ 1655689518,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,100],
 [ 1655689517,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,100],


 [ 1655689210,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,0],
 [ 1655689209,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,100],
 [ 1655689208,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,null],
 [ 1655689207,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,null],
 [ 1655689206,0,0,100,0,0,0,0,0,0,0,0,0,0,0,100,0,0,0,0,100,0,100,0,0,null],
```

Another example:

```sh
$ curl -k https://localhost:19999/api/v1/data?chart=filesystem.vfs_deleted_objects&group=countif

{
 "labels": ["time","delete"],
    "data":
 [
 [ 1655690660,null],
 [ 1655690655,0],
```